src/features/home/components/modules/video/VideoModule.tsx

```diff
@@ -95,23 +98,39 @@ export const VideoModule: FunctionComponent<VideoModuleProps> = (props) => {
           <VideoModal
             visible={videoModalVisible}
             hideModal={hideVideoModal}
-            offer={offer}
+            offers={offers}
             moduleId={props.id}
+            isMultiOffer={isMultiOffer}
             {...props}
           />
           <Spacer.Column numberOfSpaces={2} />
-          <OfferVideoModule
-            offer={offer}
-            color={props.color}
+          {!isMultiOffer && (
+            <OfferVideoModule
+              offer={offers[0]}
+              color={props.color}
+              hideModal={hideVideoModal}
+              moduleId={props.id}
+              moduleName={props.title}
+              homeEntryId={props.homeEntryId}
+              analyticsFrom={'home'}
+            />
+          )}
+        </VideoOfferContainer>
+      </StyledWrapper>
+      {!!isMultiOffer && (
```

Comment:
isMultiOffer est déjà un booléen non ? Besoin de l'opérateur !! devant ?

Reply:
En effet pas nécessaire

Reply:
On a une règle ESLint un peu naze qui nous oblige à faire ça, si je dis pas de bêtises, c'est peut-être pour ça que tu as écrit ça

Reply:
En effet la règle remonte cette erreur : `Do not use string && component, use (string ? component : null) or ( !!string && component)` même si c'est un booléen et non une string